### PR TITLE
zh: update daily-check.md

### PR DESCRIPTION
**Pull request by GMHDBJD**
### What is changed, added or deleted? (Required)
- update daily-check.md

### Which TiDB Data Migration (DM) version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/daily-check.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+category: reference
+---
+
+# Data Migration 日常巡检
+
+本文总结了 DM 工具日常巡检的方法。
+
+1. 执行 `query-status` 命令查看任务运行状态以及相关错误输出。
+
+2. 如果使用 DM-Ansible 部署 DM 集群时，正确部署了 Prometheus 与 Grafana，且 Grafana 的地址为 `172.16.10.71`，可在浏览器中打开 <http://172.16.10.71:3000> 进入 Grafana，选择 DM 的 dashboard 即可查看 DM 相关监控项。具体监控指标参照[监控与告警设置](monitor-a-dm-cluster.md)。
```
Comment (csuzhangxc) [inline zh/daily-check.md:13]: `且 Grafana 的地址为` 改成类似 `如 Grafana 的地址为`，是不是好一些？

File: zh/daily-check.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+title: Data Migration 日常巡检
+summary: 了解 DM 工具的日常巡检。
+category: reference
+---
+
+# Data Migration 日常巡检
+
+本文总结了 DM 工具日常巡检的方法。
+
+1. 执行 `query-status` 命令查看任务运行状态以及相关错误输出。
```
Comment (csuzhangxc) [inline zh/daily-check.md:11]: 链接一下到 `query-status` 吗？

Comment (TomShawn) [inline zh/daily-check.md:11]:
```suggestion
+ 方法一：执行 `query-status` 命令查看任务运行状态以及相关错误输出。
```

File: zh/daily-check.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+2. 如果使用 DM-Ansible 部署 DM 集群时，正确部署了 Prometheus 与 Grafana，且 Grafana 的地址为 `172.16.10.71`，可在浏览器中打开 <http://172.16.10.71:3000> 进入 Grafana，选择 DM 的 dashboard 即可查看 DM 相关监控项。具体监控指标参照[监控与告警设置](monitor-a-dm-cluster.md)。
+
+3. 通过日志文件查看 DM 运行状态和相关错误。
+
+    - DM-master 日志目录：通过 DM-master 进程参数 `--log-file` 设置。如果使用 DM-Ansible 部署 DM，则日志目录位于 DM-master 节点的 `{ansible deploy}/log/dm-master.log`。
+    - DM-worker 日志目录：通过 DM-worker 进程参数 `--log-file` 设置。如果使用 DM-Ansible 部署 DM，则日志目录位于 DM-worker 节点的 `{ansible deploy}/log/dm-worker.log`。
+    - dmctl 日志目录：与其二进制文件目录相同。
```
Comment (csuzhangxc) [inline zh/daily-check.md:19]: 这个感觉可以去掉，一般不用 care

File: zh/daily-check.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+---
+title: Data Migration 日常巡检
+summary: 了解 DM 工具的日常巡检。
+category: reference
+---
+
+# Data Migration 日常巡检
+
+本文总结了 DM 工具日常巡检的方法。
```
Comment (TomShawn) [inline zh/daily-check.md:9]:
```suggestion
本文总结了 DM 工具日常巡检的方法：
```

File: zh/daily-check.md (hunk 1 above)
Comment (TomShawn) [inline zh/daily-check.md:13]:
```suggestion
+ 方法二：如果使用 DM-Ansible 部署 DM 集群时正确部署了 Prometheus 与 Grafana，且 Grafana 的地址为 `172.16.10.71`，可在浏览器中打开 <http://172.16.10.71:3000> 进入 Grafana，选择 DM 的 Dashboard 即可查看 DM 相关监控项。具体监控指标参照[监控与告警设置](monitor-a-dm-cluster.md)。
```

File: zh/daily-check.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+1. 执行 `query-status` 命令查看任务运行状态以及相关错误输出。
+
+2. 如果使用 DM-Ansible 部署 DM 集群时，正确部署了 Prometheus 与 Grafana，且 Grafana 的地址为 `172.16.10.71`，可在浏览器中打开 <http://172.16.10.71:3000> 进入 Grafana，选择 DM 的 dashboard 即可查看 DM 相关监控项。具体监控指标参照[监控与告警设置](monitor-a-dm-cluster.md)。
+
+3. 通过日志文件查看 DM 运行状态和相关错误。
```
Comment (TomShawn) [inline zh/daily-check.md:15]:
```suggestion
+ 方法三：通过日志文件查看 DM 运行状态和相关错误。
```

File: zh/daily-check.md (hunk 3 above)
Comment (GMHDBJD) [reply zh/daily-check.md:19]: 是指去掉dmctl日志还是整个第三点？

Comment (csuzhangxc) [reply zh/daily-check.md:19]: dmctl 这一行

File: zh/daily-check.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+---
+title: Data Migration 日常巡检
+summary: 了解 DM 工具的日常巡检。
+category: reference
+---
+
+# Data Migration 日常巡检
+
+本文总结了 DM 工具日常巡检的方法:
```
Comment (TomShawn) [inline zh/daily-check.md:9]:
```suggestion
本文总结了 DM 工具日常巡检的方法：
```

**Comment by TomShawn** [comment]
LGTM

**Comment by TomShawn** [comment]
@miaoqingli PTAL

File: zh/daily-check.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
+---
+title: Data Migration 日常巡检
```
Comment (toutdesuite) [inline zh/daily-check.md:2]:
```suggestion
title: TiDB Data Migration 日常巡检
```

File: zh/daily-check.md [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+summary: 了解 DM 工具的日常巡检。
+category: reference
+---
+
+# Data Migration 日常巡检
```
Comment (toutdesuite) [inline zh/daily-check.md:7]:
```suggestion
# TiDB Data Migration 日常巡检
```

File: zh/daily-check.md [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+
+本文总结了 DM 工具日常巡检的方法：
```
Comment (toutdesuite) [inline zh/daily-check.md:9]:
```suggestion
本文总结了 TiDB Data Migration (DM) 工具日常巡检的方法：
```

**Comment by TomShawn** [comment]
@miaoqingli PTAL

**Comment by miaoqingli** [comment]
LGTM

**Pull request merged**